src/FortifyServiceProvider.php

```diff
@@ -87,6 +91,11 @@ public function registerPublishers(): void
             __DIR__.'/../config/fortify.php' => config_path('fortify.php'),
         ], 'config');
 
+        $this->mergeConfigFrom(
+            __DIR__.'/../config/fortify.php',
+            'fortify'
+        );
```

Comment:
what's this needed for 🤔 we could just republish the config

Reply:
If we don't merge the config file, we aren't able to use `config()` in the tests.